packages/contracts/tasks/set-owner.ts

```diff
@@ -0,0 +1,73 @@
+/* Imports: External */
+import { ethers } from 'ethers'
+import { LedgerSigner } from '@ethersproject/hardware-wallets'
+import { task } from 'hardhat/config'
+import * as types from 'hardhat/internal/core/params/argumentTypes'
+
+import { predeploys } from '../src/predeploys'
+import { getContractFactory } from '../src/contract-defs'
+
+task('set-owner')
+  .addParam('owner', 'the new oracle address', 0, types.string)
+  .addOptionalParam('transactionGasPrice', 'tx.gasPrice', undefined, types.int)
```

Comment:
if this is intended for only L2 we might want to get rid of this optional param

Reply:
We want to be able to specify `gasPrice: 0` on L2 for simplicity of usage, see the commit message for rationale